dockerfiles/rsyncd.conf

```diff
@@ -1,9 +1,12 @@
 pid file = $RSYNC_RUN_DIR/rsync.pid
 lock file = $RSYNC_RUN_DIR/rsync.lock
 log file = /dev/stdout
-
-[mnt]
-path = /mnt
 use chroot = false
 read only = false
 timeout = 300
```

Comment:
I cannot be completely sure, but at least my reading of the rsyncd.conf man page is that these 3 parameters (`use chroot`, `read only`, `timeout`) are for the modules only.  Also, this is what I saw in examples. Unless we want to go deep in testing of rsyncd, I suggest to copy them into each module individually.

Reply:
I am able to rsync to /repos with this config file.  I have also tried toggling `use chroot` and `read only` parameters to true and that causes rsync to fail.  So the global parameters are working as intended.

Reply:
Here is an example here where some module parameters are configured as global:
https://linuxconfig.org/how-to-setup-the-rsync-daemon-on-linux